test-utils/src/commonMain/kotlin/mock/focus/MockFocusHandler.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.test.mock.focus
+
+import com.splendo.kaluga.keyboard.FocusHandler
+import kotlinx.coroutines.flow.MutableStateFlow
+
+open class BaseMockFocusHandler {
+    private val _isFocused = MutableStateFlow(false)
```

Comment:
Cont a change request but an option to consider. For me it looks more like `AtomicBoolian`

Reply:
2705a80, good point